pyserini/prebuilt_index_info.py

```diff
@@ -151,5 +151,65 @@
         "documents": 4170312,
         "unique_terms": -1,
         "downloaded": False
+    },
+    "trec-covid5": {
```

Comment:
Please change to `trec-covid-r5-abstract`
Description should be "TREC-COVID Round 5: abstract index"
Add gitlab link here: https://git.uwaterloo.ca/jimmylin/cord19-indexes

We should also have:
+`trec-covid-r5-full-text` description: "TREC-COVID Round 5: full-text index"
+`trec-covid-r5-paragraph` description: "TREC-COVID Round 5: paragraph index"